### ISSUE TITLE
problems with installation of 0.0.7 on debian stretch

### ISSUE THREAD

**Issue by jkotan** [ess-dmsc/h5cpp#308]

**Comment by martukas** [comment]
It is not really required and you can run cmake with `-DCONAN=DISABLE`.
Otherwise, you cab get conan through pip.
Let me know if either of these help.

**Comment by jkotan** [comment]
Thanks. With `-DCONAN=DISABLE` it works.

**Comment by jkotan** [comment]
Is there any reason the -DCONAN=AUTO ? In 0.0.6 it is a different behavior. 
With `-DCONAN=DISABLE` creation of the debian packages is less straightforward.

**Comment by martukas** [comment]
Yes, for auto you will also need to specify `-DCONAN_FILE=conanfile_desy.txt`. I changed the behavior to have the CMake code more closely mirror what we have with our other projects at ESS. It makes it easier to maintain things when we have common code across the projects. Sorry, I know this pulls the blanket a bit in our direction, but since we are doing most of the development and testing going forward I thought was ok to be slightly selfish ;) Maybe someday our configurations will converge and then we have a common conanfile.

**Comment by jkotan** [comment]
fair enough ;-) Thanks.
